src/app/embed/adapters/feed.adapter.ts

```diff
@@ -10,83 +10,135 @@ const GUID_PREFIX = 's1!';
 @Injectable()
 export class FeedAdapter implements DataAdapter {
 
-  private feedUrl: string;
-  private feedId: string;
-  private guid: string;
+  static logError(msg: string | Error) {
+    console.error(`${msg}`);
```

Comment:
Was looking for some way to log these errors (http non-200s, xml parsing, guid-not-found, etc).  Landed on this for now.